lib/shared/bucketing-assembly-script/assembly/bucketing/bucketing.ts

```diff
@@ -141,13 +141,20 @@ function evaluateSegmentationForFeature(
     // Returns the first target for which the user passes segmentation
     for (let i = 0; i < feature.configuration.targets.length; i++) {
         const target = feature.configuration.targets[i]
+        const boundedHashData = _generateBoundedHashes(user.user_id, target._id)
+        const rolloutHash = boundedHashData.rolloutHash
```

Comment:
Can we move this inside of `if (target.rollout && passthroughRolloutEnabled) {`, so we don't generate the hash if its not enabled.

Reply:
The bounded hash is still used for bucketing

Reply:
In this function? it seems like `rolloutHash` is only ever used here in `_doesUserPassRollout()`